src/cosalib/build.py

```diff
@@ -129,8 +129,7 @@ def build_root(self):
     @property
     def build_name(self):
         """ get the name of the build """
-        ref = str(self.get_meta_key("meta", "ref")).split('-')
-        return ref[-1]
+        return str(self.get_meta_key("meta", "name")
```

Comment:
Missing a closing paren; same on the PR to master.  See https://github.com/coreos/coreos-assembler/pull/600#discussion_r302154184